Formula/y/yt-dlp.rb

```diff
@@ -72,7 +72,10 @@ class YtDlp < Formula
   end
 
   def install
-    system "gmake", "pypi-files" if build.head?
+    if build.head?
+      system "gmake", "lazy-extractors"
+      system "gmake", "pypi-files"
+    end
```

Comment:
```suggestion
    system "gmake", "lazy-extractors", "pypi-files" if build.head?
```
Isn't this equivalent?

Reply:
I thought about that but decided to go with two steps for better debugability. Maybe thats not a factor though?